libgalois/include/katana/PropertyGraph.h

```diff
@@ -66,8 +135,8 @@ struct KATANA_EXPORT GraphTopology {
     return MakeStandardRange<edge_iterator>(begin_edge, end_edge);
   }
   Node edge_dest(Edge eid) const {
-    KATANA_LOG_ASSERT(eid < static_cast<Edge>(out_dests->length()));
-    return out_dests->Value(eid);
+    KATANA_LOG_DEBUG_ASSERT(eid < dests_.size());
```

Comment:
How about using `edge_id` instead of `eid? like `node_id`?

Reply:
done